code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -188,227 +185,173 @@
 			if(!key)
-				msg += "[span_deadsay("[t_He] [t_is] totally catatonic. The stresses of life in deep-space must have been too much for [t_him]. Any recovery is unlikely.")]\n"
+				. += span_deadsay("<br>[t_He] [t_is] totally catatonic. The stresses of life in deep-space must have been too much for [t_him]. Any recovery is unlikely.")
 			else if(!client)
-				msg += "[span_deadsay("[t_He] [t_has] a blank, absent-minded stare and appears completely unresponsive to anything. [t_He] may snap out of it soon.")]\n"
+				. += span_deadsay("<br>[t_He] [t_has] a blank, absent-minded stare and appears completely unresponsive to anything. [t_He] may snap out of it soon.")
 
 	var/scar_severity = 0
-	for(var/i in all_scars)
-		var/datum/scar/S = i
-		if(S.is_visible(user))
-			scar_severity += S.severity
-
-	switch(scar_severity)
-		if(1 to 4)
-			msg += "[span_tinynoticeital("[t_He] [t_has] visible scarring, you can look again to take a closer look...")]\n"
-		if(5 to 8)
-			msg += "[span_smallnoticeital("[t_He] [t_has] several bad scars, you can look again to take a closer look...")]\n"
-		if(9 to 11)
-			msg += "[span_notice("<i>[t_He] [t_has] significantly disfiguring scarring, you can look again to take a closer look...</i>")]\n"
-		if(12 to INFINITY)
-			msg += "[span_notice("<b><i>[t_He] [t_is] just absolutely fucked up, you can look again to take a closer look...</i></b>")]\n"
-	msg += "</span>" // closes info class
-
-	if (length(msg))
-		. += span_warning("[msg.Join("")]")
-
-	var/trait_exam = common_trait_examine()
-	if (!isnull(trait_exam))
+	for(var/datum/scar/scar as anything in all_scars)
+		if(scar.is_visible(user))
+			scar_severity += scar.severity
+
+	if(scar_severity >= 1)
+		. += "" // since this is a more of a chaser, give it space
+		switch(scar_severity)
+			if(1 to 4)
+				. += span_tinynoticeital("[t_He] [t_has] visible scarring, you can look again to take a closer look...")
+			if(5 to 8)
+				. += span_smallnoticeital("[t_He] [t_has] several bad scars, you can look again to take a closer look...")
+			if(9 to 11)
+				. += span_notice("<i>[t_He] [t_has] significantly disfiguring scarring, you can look again to take a closer look...</i>")
+			if(12 to INFINITY)
+				. += span_notice("<b><i>[t_He] [t_is] just absolutely fucked up, you can look again to take a closer look...</i></b>")
+
+	var/trait_exam = common_trait_examine(user)
+	if (length(trait_exam))
 		. += trait_exam
 
-	if(isliving(user))
-		var/mob/living/privacy_invader = user
-		if(HAS_MIND_TRAIT(privacy_invader, TRAIT_MORBID))
-			if(HAS_TRAIT(src, TRAIT_DISSECTED))
-				msg += "[span_notice("[t_He] appears to have been dissected. Useless for examination... <b><i>for now.</i></b>")]\n"
-			if(HAS_TRAIT(src, TRAIT_SURGICALLY_ANALYZED))
-				msg += "[span_notice("A skilled hand has mapped this one's internal intricacies. It will be far easier to perform future experimentations upon [t_him]. <b><i>Exquisite.</i></b>")]\n"
```

Comment:
And also stuff like this code which was dead code because `msg` was already added to `.` so this was doing nothing